tools/linux/maple_upload

```diff
@@ -28,6 +28,12 @@ DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 "${DIR}/upload-reset" ${dummy_port_fullpath} 750
 
+echo "Waiting for device in DFU mode"
+while ! lsusb | grep -i ${usbID}
+do
+  sleep 1
+  echo "."
```

Comment:
I'd suggest doing `echo -n .` instead, i.e. remove the newline. Also, add an empty echo after `done`, so it will look like this:
```
Waiting for device in DFU mode .........
```